apps/app/actions/action.tsx

```
@@ -0,0 +1,156 @@
+      await sleep(1000);
+      if (done) {
+        history.done([...history.get(), { role: "assistant", content }]);
+      }
+      return <BotMessage>{content}</BotMessage>;
+    },
+    tools: {
+      workspace: {
+        description:
+          "when the user responds with yes then render the workspace form",
+        parameters: z.object({}),
+        generate: async function* ({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)